### ISSUE TITLE
this is dark side hack index

### ISSUE THREAD

**Issue by Prince8791** [mozilla/Campus-Activities#27]
<html>
<head>
	<title>DARK SIDE</title>
	<meta charset="utf-8" />
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
	<script src="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.js"></script>
	<script src="osmtogejson.js"></script>
	<script src="leaflet-osm.js"></script>
	<script src="http://maps.googleapis.com/maps/api/js?v=3.exp&signed in=true&libraries=places"></script>
	<script src="google-maps-search.js"></script>
	<script src="slider/jquery.nouislider.min.js"></script>
	<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.5/js/materialize.min.js"></script>
	<script src="main.js"></script>
  	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.5/css/materialize.min.css">
    <link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
	<link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css" />
    <link rel="stylesheet" href="slider/jquery.nouislider.min.css" />
    <link rel="stylesheet" href="css/stylesheet.css" />

</head>
<body>
	<div id="intro" class="light-blue darken-2" style="width: 100%; height: auto; background-color: #2980b9;">

			<div class="illustration">
			  	<img src="img/illustration.svg" alt="Water Illustration">
			</div>

    		<div class="maininfo row">
      			<h1>maintaining <mark>water points</mark> in communities </h1>
    		</div>

			<div class= "searchbar row">
			    <div class="col s12">
				        <div class="col s11"> <input id="txtautocomplete2" type="text" placeholder="Search location"> </div>
				        <div class="col s1"> <a href="#" onclick="skipIntro();" class="waves-effect waves-light btn  blue-grey lighten-5 material-icons">search</a> </div>

      			</div>
   			</div>
	</div>

	<a class="modal-trigger waves-effect waves-light" style="position: absolute; bottom: 20px; right: 20px; " href="#popup1"><i class="medium material-icons" style="color: white; font-size: 2.5rem">message</i></a>
		<div id="popup1" class="modal">
		  <div class="modal-content">
		    <h3>Volunteers helping map water points <br> in communities across the world</h3>
		    <p>
		      Knowing the location of water points across the world, we are able to maintain them, <br>reducing Cholera infectons in vulnerable communities. This map shows the impact made by contributers so far. Search a specific location and drag the timescale along to view the water points plotted.
		    </p>

		   <div id="key">
		   	<div class="row">
		   		<div class="col s5 " id="waterkey"> <img class="keyicon" src="img/droplet.png"> <p> shows water <br>sources</p></div>
		   		<div class="col s5"> <img class="keyicon bluearea" src="img/Blue_area.png"> <p> shows other <br>mapped areas</p></div>
			</div>
			</div>

				<div class="logo">
				  	<a href="http://www.msf.org/"><img src="http://cdn.msf.org/sites/msf.org/files/msf_international_logo_colour_rgb-215x93xpng24.png" alt="MSF Logo"></a>
				  	<a href="http://www.missingmaps.org/"><img src="http://wiki.openstreetmap.org/w/images/a/a3/Missing-Maps-logo.jpg" alt="Missing Maps"></a>
				</div>
			</div>
			 
		 </div>

	<div id="main">
		<nav id="mynavbar" class="light-blue darken-2">
		  <div class="nav-wrapper">
		  	<ul class="center brand-logo">
		    <li id="render-button">
		    	
			   <button id="init-button" onclick="Materialize.toast('Downloading data...', 2000); init();" class="blue-grey lighten-5 btn waves-effect waves-light" type="submit" name="action" >Show timeline
	 			</button>
	 		</li>
	 		<li>
	 			<div id="render-warn">
					<b>Zoom closer to view timeline.</b>
				</div>
			</li>
			</ul>

			<ul class="right" id="render-section" style="padding-right: 30px; padding-top: 2px;">
				<li class="slider-details" style="font-size: 1.2rem;display: none;" id="date_field"></li>
			</ul>

		      <div class="left" style="width:400px;">
		        <div class="input-field">
		          <input id="txtautocomplete" placeholder="Search for places" type="search" required>
		          <label for="search"><i class="material-icons">search</i></label>
		          <i class="material-icons">close</i>
		        </div>
		      </div>

		  </div>
	  	</nav>

	  	<div style="position: absolute;z-index: -1;" id="map"></div>

	  	 <div id="myloader" style="position: absolute; left: 0px; top: 0px;z-index: 2;display: none;" class="progress"> <div class="indeterminate light-blue darken-3"></div> </div>

	  	<div class="slider-details" style="display: none;"><div style="box-shadow: 1px 1px 5px 0px #888888; position: absolute; right: 100px; top: 80px; width: 300px;" id="slider"></div></div> 

	  	<div class="slider-details" id="animation-button" onclick="playAnimation();" style="position: absolute; right: 30px; top: 10px;display: none;"><a class="btn-floating btn-large waves-effect waves-light light-blue darken-2" style="width: 35px; height: 35px; line-height: 49px">
				<i id="player-b" class="material-icons" style="line-height: 6px; font-size: 1.6rem">play_arrow</i></a></div>
	</div>

	</div>

</body>
</html>
